lib/AsyncStorage.js

```diff
@@ -100,6 +100,11 @@ const AsyncStorage = {
   ): Promise<null> {
     return new Promise((resolve, reject) => {
       RCTAsyncStorage.multiSet([[key, value]], function(errors) {
+        if (Platform.OS === 'ios' && typeof value !== 'string') {
```

Comment:
Any reason why we only do this on iOS?

Reply:
Just checked against android - this this warning should also be shown.

Reply:
Also: on `multiSet` should also display warning